doc/user-guide/push-notifications/With-MongoosePush.md

```diff
@@ -0,0 +1,88 @@
+## Push notifications with MongoosePush
+
+MongoosePush is a simple RESTful service written in Elixir. It provides the ability to send push
+notifications to FCM (Firebase Cloud Messaging) and/or APNS (Apple Push Notification Service) via
+their HTTP/2 API.
+
+To take advantage of MongoosePush's functionality, you will need to enable the
+`mod_push_service_mongoosepush` module: this module acts as a bridge between the
+`push_notifications` hook and [MongoosePush][] itself.
+
+### Getting started
+
+To enable integration with MongoosePush, it is as simple as the next two steps.
+First, you need to define a pool of HTTPS connections to MongoosePush in the
+`outgoing_pools` section:
+
+```Erlang
+{outgoing_pools, [
+    (...)
+    {http, global, mongoose_push_http,
+        [{strategy, available_worker}],
+        [{server, "https://localhost:8443"}]},
+    (...)
+    ]
+}.
+```
+
+And second, you need to add `mod_push_service_mongoosepush` to the `modules` section in the config file:
+
+```Erlang
+{modules, [
+    (...)
+    {mod_push_service_mongoosepush, [
+        {pool_name, mongoose_push_http},
+        {api_version, "v3"}]},
+    (...)
+    ]
+}.
+```
+
+Here, we assume that [MongoosePush][] will be available on the localhost on port 8443, which is the
+default one — note the `server` option in the outgoing pool definition.
+Next we enable [mod_push_service_mongoosepush][]. The first option is the name of the HTTP pool to
```

Comment:
Looks like the `mod_push_service_mongoosepush` link is not defined. When viewing the file on github (https://github.com/esl/MongooseIM/blob/docs/push_notifications/doc/user-guide/push-notifications/With-MongoosePush.md) I can see the raw text here instead of hyperlink.